src/web/modules/stripe/index.js

```diff
@@ -1,6 +1,6 @@
 // @TODO(sfount) improve TS export -> JS import
-const httpBasic = require('./basic/basic.http').default
-const httpBasicTest = require('./basic/test-account.http').default
+const httpBasic = require('./stripe-account.http.').default
```

Comment:
extra dot at the end?
```suggestion
const httpBasic = require('./stripe-account.http').default
```

Reply:
Good spot, have fixed